tests/MassTransit.ActiveMqTransport.Tests/DelayRetry_Specs.cs

```diff
@@ -238,8 +263,16 @@ public async Task Should_properly_redeliver(string flavor)
                     });
                     cfg.EnableArtemisCompatibility();
                 }
+                else if (flavor == ActiveMqHostAddress.AmqpScheme)
```

Comment:
Seems like a 'ConfigureTestHost` extension or something is needed to avoid all this duplication?